### PR TITLE
[PBW-5174] Control bar overhaul part 1 of 2

### PR DESCRIPTION
**Pull request by Jgkelle26**
Please do not merge with out checking out part 2 skin.json

### CODE REVIEW

**Comment by Jgkelle26** [comment]
Part 2 of changes: https://github.com/ooyala/skin-config/pull/135

**Comment by rchalooyala** [comment]
Part 3: https://github.com/ooyala/skin-config/pull/136

File: scss/responsive/_small.scss [hunk 1]
```diff
@@ -1,5 +1,13 @@
 .oo-small {
 
+  // Control bar
+    .oo-control-bar-items-wrapper {
+      .oo-control-bar-item {
+        padding: 0 14.5px * $xsmall-multiple;
+        font-size: 23px;
+       }
+      }
+
```
Comment (rchalooyala) [inline scss/responsive/_small.scss]: formatting

File: scss/components/_control-bar.scss [hunk 2]
```diff
@@ -167,3 +184,6 @@ a.oo-volume-bar {
   height: 0 !important;
   opacity: 0;
 }
+
+
+
```
Comment (rchalooyala) [inline scss/components/_control-bar.scss]: no need for these spaces

File: scss/components/_control-bar.scss [hunk 3]
```diff
@@ -118,32 +121,46 @@
...
+    position: relative;
+    width: 9px;
+    height: 8px;
+    padding-left: 7px;
+    background-color: rgba($white, 0.6);
+    @include vendor-prefixes (transition, all .18s ease-in-out);
+}
+
+.oo-volume-bar:hover{
+height: 17px;
```
Comment (rchalooyala) [inline scss/components/_control-bar.scss]: formatting

File: scss/components/_control-bar.scss [hunk 4]
```diff
@@ -118,32 +121,46 @@
...
+
 .oo-volume-bar {
-  display: inline-block;
-  background-clip: content-box;
-  position: relative;
-  width: 6px;
-  height: 18px;
-  padding-left: 3px;
-  background-color: rgba($white, 0.6);
+    margin-bottom: 4px;
```
Comment (rchalooyala) [inline scss/components/_control-bar.scss]: formatting

File: scss/responsive/_xsmall.scss [hunk 5]
```diff
@@ -178,19 +178,22 @@
...
+    height: $control-bar-height-small;
+    font-size: 25px * $xsmall-multiple;
 
     .oo-control-bar-items-wrapper {
       .oo-control-bar-item {
-        font-size: 18px * $xsmall-multiple;
-        padding: 0 7.5px * $xsmall-multiple;
+        font-size: 25px * $xsmall-multiple;
+        padding: 0 8.5px * $xsmall-multiple;
+    margin-top: 2px;
```
Comment (rchalooyala) [inline scss/responsive/_xsmall.scss]: formatting

File: scss/responsive/_xsmall.scss [hunk 6]
```diff
@@ -178,19 +178,22 @@
...
       }
 
       .oo-control-bar-duration {
         font-size: 14px * $xsmall-multiple;
         padding-left: 7.5px * $xsmall-multiple;
         padding-right: 7.5px * $xsmall-multiple;
+        margin-top: 6px;
+
```
Comment (rchalooyala) [inline scss/responsive/_xsmall.scss]: space not needed

File: scss/components/_control-bar.scss [hunk 7]
```diff
@@ -118,32 +121,46 @@
...
 }
 
-.oo-volume-bar.oo-on {
-  background-color: rgba($blue-light, 0.6);
+.oo-on{
+  background-color: rgba($blue-light, 1.0);
+  height: 17px;
+  padding-bottom: 0px;
+  margin-bottom: 4px;
+
```
Comment (rchalooyala) [inline scss/components/_control-bar.scss]: extra line

**Pull request merged**